### PR TITLE
Add gravity attribute.

### PR DESCRIPTION
**Pull request by akihiro0228**
Add gravity attribute. (Gravity is Android-like)
Types are Top, Center, and Bottom.

### CODE REVIEW

**Comment by pine613** [comment]
:+1:

**Comment by mattak** [comment]
:+1:

**Comment by karupanerura** [comment]
:+1:

File: JLToast/JLToastView.swift [hunk 1]
```diff
@@ -26,6 +26,13 @@ public let JLToastViewTextColorAttributeName = "JLToastViewTextColorAttributeNam
 public let JLToastViewFontAttributeName = "JLToastViewFontAttributeName"
 public let JLToastViewPortraitOffsetYAttributeName = "JLToastViewPortraitOffsetYAttributeName"
 public let JLToastViewLandscapeOffsetYAttributeName = "JLToastViewLandscapeOffsetYAttributeName"
+public let JLToastViewGravityAttributeName = "JLToastViewGravityAttributeName"
+
+public struct JLToastGravity {
```
Comment (devxoul) [inline JLToast/JLToastView.swift]: I think we can define `JLToastGravity` with `enum`. Is this intended?
If you had considered Objective-C compatible, it should be defined in `JLToast.h`, too.

**Comment by devxoul** [comment]
I think we should align toasts for `.Center`.

toast

**Pull request closed without merging**